File: Practice/src/Prime.java
import java.util.Scanner;
public class Prime {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.println("Enter a number: ");
		int n = scan.nextInt();
		System.out.println("The first " + n + " prime numbers are: ");
		
		int count =0;
		int number =2;
		while(count<n) {
			if(isPrime(number)) {
				System.out.println(number+" ");
				count++;
			}
			number++;
		}

	}
	
	public static boolean isPrime(int num) {
		if(num<=1) {
			return false;
		}
		for(int i=2;i<=num/2;i++) {
			if(num%i==0) {
				return false;
			}
		}
		return true;
	}
}

File: Preparation_Python/2025/Prime.py
n = int(input("Enter a number"))

if n <= 1:
    print("Not Prime")
else:
    i=2
    isPrime = True 
    while(i * i <= n):
        if(n % i ==0):
            isPrime = False
            break
        i += 1
    print("Prime" if isPrime else "Not prime")

    